src/test/Components/CreateImageWizardV2/CreateImageWizard.compliance.test.tsx

```diff
@@ -154,6 +154,13 @@ describe('Step Compliance', () => {
         /cis red hat enterprise linux 8 benchmark for level 1 - workstation/i
       )
     );
+    screen.getByText(/kernel arguments:/i);
+    screen.getByText(/audit_backlog_limit=8192 audit=1/i);
+    screen.getByText(/disabled services:/i);
+    screen.getByText(/nfs-server/i);
+    screen.getByText(/enabled services:/i);
+    screen.getByText(/enabled services:/i);
```

Comment:
Should this be a list of the enabled services instead of a duplicate of the previous line?

Reply:
sure, you correct :)